testing/trino-product-tests/src/main/java/io/trino/tests/product/TestFunctions.java

```diff
@@ -45,4 +45,20 @@ public void testVersion()
                 onTrino().executeQuery("SELECT version()").getOnlyValue(),
                 onTrino().executeQuery("SELECT node_version FROM system.runtime.nodes WHERE coordinator = TRUE").getOnlyValue());
     }
+
+    @Test(groups = FUNCTIONS)
```

Comment:
We don't need a product test for this. It's covered in the engine tests above.

Product tests are more expensive to run and harder to work with, so they should only be limited to testing full-system integrations that cannot be replicated within unit tests or with testcontainers.

Reply:
Also note that the existing engine tests effectively run on multiple nodes via DistributedQueryRunner.

Reply:
What would be the quickest solution to make those engine tests run via DistributedQueryRunner ?